### PR TITLE
adding rm_strat

### PR DESCRIPTION
**Pull request by annelle-abatoni**
Adding function with documentation

### CODE REVIEW

File: test/runtests.jl [hunk 1]
```diff
@@ -61,3 +61,6 @@ end
 # end
 
 
+table = CSV.read("test/files/humann2:SRS014459-Stool_genefamilies.tsv", DataFrame, delim='\t',
+    header=["#GeneFamily", "SRS014459-Stool_Abundance"], datarow = 2
```
Comment (kescobo) [inline test/runtests.jl:65]: Need closing parens here. Please try to make sure tests pass when you push

File: src/general.jl [hunk 2]
```diff
@@ -41,8 +41,34 @@ function clean_abundance_tables(files::Array{String, 1};
...
+1×2 DataFrame
+ Row │ #GeneFamily                        SRS014459-Stool_Abundance 
+     │ String                             Float64                   
+─────┼──────────────────────────────────────────────────────────────
+   1 │ UniRef50_unknown                                     69.0221
+```
+"""
 function rm_strat!(df::DataFrame; col::Union{Int, Symbol}=1)
-    filter!(row->!occursin(r"\|", row[1]), df)
+    table=filter!(row->!occursin(r"\|", row[1]), df)
```
Comment (kescobo) [inline src/general.jl:71]: There's no sense in assigning a variable here, you never use it later

File: src/general.jl [hunk 3]
```diff
@@ -41,8 +41,34 @@ function clean_abundance_tables(files::Array{String, 1};
...
 
 
+"""
+    rm_strat!(df::DataFrame; col::Union{Int, Symbol}=1)
+
+Given an abundance table of functional profiles ex. humann2 file, removes stratification to show total abundances.
+```jldoctest rm_strat!
+Examples
+≡≡≡≡≡≡≡≡≡≡
+julia> table = CSV.read("test/files/humanntestfile.tsv", DataFrame)
```
Comment (kescobo) [inline src/general.jl:51]: This is great, but it doesn't look like this file got added / committed

File: src/general.jl [hunk 4]
```diff
@@ -41,8 +41,34 @@ function clean_abundance_tables(files::Array{String, 1};
 end
 
 
+"""
+    rm_strat!(df::DataFrame; col::Union{Int, Symbol}=1)
+
+Given an abundance table of functional profiles ex. humann2 file, removes stratification to show total abundances.
+```jldoctest rm_strat!
+Examples
```
Comment (kescobo) [inline src/general.jl:49]: This needs to go outside the code block

**Review by kescobo — CHANGES_REQUESTED** [review]

File: src/general.jl (hunk 3 above)
Comment (annelle-abatoni) [reply src/general.jl:51]: I just added it! I though it got added when I did commit -am ... but I am assuming new files need to get added explicitly?

**Comment by kescobo** [comment]
Gonna close this, since we'll be removing DataFrames-based functions, and since there's a more generic way to do this via Microbiome.jl

```
julia> features(comm)
3-element Vector{GeneFunction}:
 GeneFunction("gene1", Taxon("tax1", :species))
 GeneFunction("gene1", Taxon("tax2", :genus))
 GeneFunction("gene2", missing)

julia> features(filter(hastaxon, comm))
2-element Vector{GeneFunction}:
 GeneFunction("gene1", Taxon("tax1", :species))
 GeneFunction("gene1", Taxon("tax2", :genus))
```

**Pull request merged**